_posts/2015-05-06-gui.md

```diff
@@ -0,0 +1,12 @@
+---
+title:  "Mozregression GUI is out!"
+author: Julien Pagès
+---
+
+I am proud to announce the first release of **mozregression-gui**, a
+graphical interface over the command line tool!
```

Comment:
a graphical interface over the command line tool! -> a graphical interface for mozregression!